integration/shortlived_pods_test.go

```diff
@@ -21,7 +21,6 @@ func TestShortlivedPods(t *testing.T) {
 	defer cancel()
 	f.WaitForAllPodsInPhase(ctx, "app=shortlived-pod-fail", v1.PodFailed)
 	f.WaitForAllPodsInPhase(ctx, "app=shortlived-pod-success", v1.PodSucceeded)
-	f.KillProcs()
```

Comment:
?

Reply:
Test teardown kills them anyway, but first tries to dump engine state. This was killing them early, so when the test failed, you wouldn't get any useful output. There's no reason afaict to kill it before asserting - I think it's just an old test that might pre-date the `TearDown()`.

(This test was legitimately failing at one point because I broke podlogstream and it was annoying to debug 😓 )